### DISCUSSION TITLE
A axis (rotary) speed is not defined correctly

### DISCUSSION THREAD

**Discussion by MadaStanescu** [community/community#68339] [Other Feature Feedback, Questions, & Ideas]
![IMG_20230926_094133](https://github.com/community/community/assets/61333859/e94e857b-1ba8-4cc0-ba7c-275fbd84909a)
![IMG20230926100440](https://github.com/community/community/assets/61333859/7832a0bf-df97-4e29-a1e2-a4aa42423448)
![IMG20230926005735](https://github.com/community/community/assets/61333859/5bf053a0-6ad7-4f36-b7c7-2d0b03c815c3)

Bug

### Body

Hello, I have problems setting the Rotary A axis. The completion time of a project is higher in A axis than if I use Y axis. Why? The whole subject of my discussion with Neje is in the link below. I have a Neje max 4 (5 axis) with the A axis. If I set the A axis with the measurement units (picture 1) from the setting Lightburn it behaves strangely (picture 2) . If I do the same thing with the Y axis everything is normal (picture 3) . Execution time of the A axis is excessively large compared to the Y axis. In the test I used the same project same dimensions for both axes. Thanks in advance.

[https://forum.neje99.com/t/max4-board-a-ex-rotary-conector/6897/69 1](https://forum.neje99.com/t/max4-board-a-ex-rotary-conector/6897/69)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @MadaStanescu ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

I've clicked that link and it is only viewable if you have an account which I'm not sure if a lot of would have that - if you could share more details that can help in getting you a response.

**Reply by MadaStanescu** [reply]
hello, I'm still waiting for your answer

**Reply by queenofcorgis** [reply]
Looking at these screenshots, I'm not sure if they are super relevant to GitHub.  It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

**Comment by MadaStanescu** [comment]
Sorry...I messed up the forum ;)
